doc/contributing/testing_ruby.md

```diff
@@ -40,7 +47,13 @@ We can run any of the make scripts [in parallel](building_ruby.md#label-Running+
     make test-all
     ```
 
-    We can run a specific test file in this suite using the `TESTS` environment variable, for example:
+    We can run a specific test directory in this suite using the `TESTS` environment variable, for example:
```

Comment:
Accurately, this is not an environment variable.

Reply:
```suggestion
    We can run a specific test directory in this suite using the `TESTS` option, for example:
```